docs/deepops/configuration.md

```diff
@@ -0,0 +1,219 @@
+
+If you're interested in customizing the behavior of a particular Ansible role further, you may also want to check the `defaults` directory for that role.
+This contains the default values of the variables included with the role itself, and may contain variables that aren't listed in the example configuration.
+However, you can override these values by adding them to your DeepOps configuration files.
+
+
+### Which file should my variables go in?
+
+**Group variables** are applied to all the hosts in an Ansible inventory group.
+These variables are specified in files named for the inventory group used, in the `config/group_vars` directory:
+
+```
+group_vars/
+├── all.yml
+├── k8s-cluster.yml
+├── netapp-trident.yml
+└── slurm-cluster.yml
+```
+
+So, for example, variables in the `all.yml` file will apply to all the hosts in your inventory;
+but variables in the `slurm-cluster.yml` file will only be used for hosts in the `slurm-cluster` inventory group.
+If you add more inventory groups, you can add files to `group_vars` to apply variables to them.
+
+**Host variables** are variables that should only be used for a specific host.
+This kind of variable should be used for parameters that vary across each machine,
+such as IP addresses,
+or test configurations,
+such as installing a different version of the NVIDIA driver on a host where you're testing an upgrade.
+These variables go in files named for each host in the `host_vars` directory:
+
+```
+host_vars/
+└── example01
+```
+
+The variables in `host_vars/example01` would only be applied for a host named `example01`.
+
+
+## Adding custom playbooks
+
+In the process of setting up your cluster, you may want to write your own Ansible playbooks for features that DeepOps doesn't include!
+
+For example, DeepOps doesn't (yet) include a playbook for installing the classic Linux game [Pingus](https://pingus.seul.org/).
+If you wrote an Ansible playbook for installing Pingus on your cluster (to give your users a way to [entertain themselves while their code compiles](https://xkcd.com/303/)),
+you would put it in your `config/playbooks` directory:
+
+```
+playbooks/
+└── pingus.yml
+```
+
+This playbook can then be run with Ansible like any other DeepOps playbook:
+
+```
+ansible-playbook config/playbooks/pingus.yml
+```
+
+And can make use of variables in your `config` directory like other playboks.
+
+
+## Managing your configuration directory in Git
+
+We recommend creating a separate Git repository for managing your cluster configuration, so that you can track changes to your cluster independently of DeepOps changes and upgrades.
+(The DeepOps [`.gitignore` file](../../.gitignore) is set up to faciliate this by ignoring directories that start with the string `config`, except for `config.example`.)
+
+A good practice is to start a new Git repository by copying the example configuration:
+
+```
+$ cp -R config.example/ config.my-cluster/
+$ cd config.my-cluster/
```

Comment:
Why are we suggesting they use config.my-cluster instead of just the config cluster that we create?

Reply:
Some old draft language sneaking in here, will fix